libhoney/src/test/java/io/honeycomb/libhoney/utils/ObjectUtilsTest.java

```diff
@@ -12,15 +12,21 @@
 
     @Test
     public void checkThatTheFormatterCanHandleZuluTime() throws ParseException {
-        final Date parse = ObjectUtils.getRFC3339DateTimeFormatter().parse("2018-05-01T12:01:00.0925Z");
+        final Date parse = ObjectUtils.getRFC3339DateTimeFormatter().parse("2018-05-01T12:01:00.925Z");
 
+        assertThat(parse).hasMinute(01); 
+        assertThat(parse).hasSecond(00);
+        assertThat(parse).hasMillisecond(925);
         assertThat(parse).isNotNull();
     }
 
     @Test
     public void checkThatTheFormatterCanHandleZonedTime() throws ParseException {
-        final Date parse = ObjectUtils.getRFC3339DateTimeFormatter().parse("2018-05-01T12:01:00.0925+01:00");
+        final Date parse = ObjectUtils.getRFC3339DateTimeFormatter().parse("2018-05-01T12:01:00.925+01:00");
 
+        assertThat(parse).hasMinute(01); 
```

Comment:
for completeness, is it worth putting in an assertion on the year, month, day too?

Reply:
ooh, and time zone?